src/components/Toggle/Toggle.css

```diff
@@ -63,19 +68,18 @@ button[category='neutral']:focus {
 
 label {
   height: 100%;
-  width: 50%;
```

Comment:
Updates the CSS rules to dynamically set the width for each label, since now we can have more than 2 options.